index.d.ts

```diff
@@ -80,6 +81,11 @@ type CommonProps<ItemType> = {
    * When set to true the library will calcualte the total dimensions taking into account padding in style prop, and padding + maxWidth/maxHeight in contentContainerStyle prop
    */
    adjustGridToStyles?: boolean;
+
+  /**
+   * Replaces default interface in FlatGrid and SectionGrid with custom interface
+   */
+   customInterface?: typeof Animated.FlatList | typeof Animated.SectionList;
```

Comment:
I don't think I have a better idea for the types. But would  it be better if we keep the type as `typeof FlatList` and `typeof SectionList`? Assuming `Animated.FlatList` has same type as that of `FlatList`